README.md

```diff
@@ -13,3 +13,159 @@
 
 GraphQL Service runs on port 23431 at `/graphql` by default
 
+
+## Queries
+Here are some of the important GraphQL queries:
+
+### 1. Get all traces in provided time range
+
+```graphql
+curl -s localhost:2020/graphql -H 'Content-Type: application/graphql' -d\
+'{
+  traces(
+    type: API_TRACE
+    limit: 10
```

Comment:
Why the limit, when we want to say get all traces?

Reply:
Can add `offset: 0` though, just like for other queries for SERVICE and API

Reply:
We have to put large time range now considering we don't know when someone is going to try it. so this is to avoid lot of junk as output. Showing few of them proves our point?

Reply:
cool. makes sense. I was just stumped by the title, which says "Get all traces", but then we were limiting by 10